hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainer.java

```diff
@@ -159,6 +159,8 @@ public void create(VolumeSet volumeSet, VolumeChoosingPolicy
         HddsVolume containerVolume;
         try {
           containerVolume = volumeChoosingPolicy.chooseVolume(volumes, maxSize);
+          // commit bytes have been reserved in volumeChoosingPolicy#chooseVolume
+          containerData.setCommittedSpace(true);
```

Comment:
@peterxcli , need move L174 & L177 before setCommittedSpace, otherwise, releaseCommitSpace will throw NPE for getVolume().